cypress/e2e/inventory/search/search-by-alternative-title-of-linked-record.cy.js

```diff
@@ -0,0 +1,125 @@
+import MarcAuthorities from '../../../support/fragments/marcAuthority/marcAuthorities';
+import QuickMarcEditor from '../../../support/fragments/quickMarcEditor';
+import MarcAuthority from '../../../support/fragments/marcAuthority/marcAuthority';
+
+const testData = {
+  user: {},
+  instanceIDs: [],
+  authorityIDs: [],
+  tags: ['130', '240'],
+  instanceRecords: [
+    'Prayer Bible (Test record with 130 linked field).',
+    'Prayer Bible (Test record with 240 linked field).',
+  ],
+  searchQueries: ['Bible. Polish. Biblia Płocka 1992', 'Abraham, Angela, 1958- Hosanna Bible'],
+  alternativeTitles: ['Biblia Płocka', 'Hosanna Bible'],
+  searchResults: [
+    'Prayer Bible (Test record with 130 linked field).',
+    'Prayer Bible (Test record with 240 linked field).',
+    'Prayer Bible (Test record without linked field: 246).',
+    'Prayer Bible (Test record without linked field: 270).',
+  ],
+  marcFiles: [
+    {
+      marc: 'marcBibC375255.mrc',
+      fileName: `testMarcFileC375255.${randomFourDigitNumber()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      numberOfRecords: 4,
+    },
+    {
+      marc: 'marcAuth130C375255.mrc',
+      fileName: `testMarcFileAuth130C375255.${randomFourDigitNumber()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numberOfRecords: 1,
+    },
+    {
+      marc: 'marcAuth240C375255.mrc',
+      fileName: `testMarcFileAuth240C375255.${randomFourDigitNumber()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numberOfRecords: 1,
+    },
+  ],
+};
+
+describe('Inventory', () => {
+  describe('Search in Inventory', () => {
+    before('Create test data', () => {
+      cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
+        testData.marcFiles.forEach((marcFile) => {
+          DataImport.verifyUploadState();
+          DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+          JobProfiles.search(marcFile.jobProfileToRun);
+          JobProfiles.runImportFile();
+          JobProfiles.waitFileIsImported(marcFile.fileName);
+          Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED);
+          Logs.openFileDetails(marcFile.fileName);
+          for (let i = 0; i < marcFile.numberOfRecords; i++) {
+            Logs.getCreatedItemsID(i).then((link) => {
+              if (marcFile.jobProfileToRun === 'Default - Create instance and SRS MARC Bib') {
+                testData.instanceIDs.push(link.split('/')[5]);
+              } else {
+                testData.authorityIDs.push(link.split('/')[5]);
+              }
+            });
+          }
+          cy.visit(TopMenu.dataImportPath);
+        });
+      });
+      cy.createTempUser([Permissions.inventoryAll.gui]).then((userProperties) => {
+        testData.user = userProperties;
+        cy.loginAsAdmin({
```

Comment:
Don't you need to log in with a created user here?

Reply:
For preconditions I need admin login. Removed redundant adminLogin